x-pack/plugins/session_view/public/components/ProcessTree/index.tsx

```diff
@@ -121,6 +128,16 @@ export const ProcessTree = ({
           onProcessSelected={onProcessSelected}
         />
       )}
+      {orphans.forEach((process) => {
```

Comment:
nitpick: `.map` just to be consistent with `ProcessTreeNode` and other components

Reply:
oh ya that is clearly probably not working haha.